src/include/platform/FailSafeContext.h

```diff
@@ -87,8 +80,16 @@ class FailSafeContext
 
     // TODO:: Track the state of what was mutated during fail-safe.
 
+    static constexpr size_t FailSafeContextTLVMaxSize()
+    {
+        return TLV::EstimateStructOverhead(sizeof(FabricIndex), sizeof(bool), sizeof(bool));
+    }
+
     static void HandleArmFailSafe(System::Layer * layer, void * aAppState);
+    static void HandleDisarmFailSafe(intptr_t arg);
```

Comment:
The difference between this and DisarmFailSafe needs to be clearly documented.  Assuming there is a difference in terms of what they do.